src/lib.rs

```diff
@@ -0,0 +1,65 @@
+/*
+ copyright: (c) 2013-2018 by Blockstack PBC, a public benefit corporation.
```

Comment:
2019

Reply:
Addressed in 18eb62befc97b271c7b99c3835298d1763245be7